active_directory_ldap/help.md

```diff
@@ -19,20 +19,25 @@
 
 ## Setup
 
-To connect, you must have NTLM credentials.
-
-Please make sure you enter your credentials with the DOMAIN\username format.
+The connection configuration accepts the following parameters:
 
-The hostname should start with the URI prefix of `ldap://` for an unencrypted connection or `ldaps://` for an encrypted connection using SSL.
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|host|string|None|True|Server Host, e.g. ldap://198.51.100.100. Must use either ldap:// or ldaps:// for SSL prefix|None|ldaps://198.51.100.100|
+|port|integer|389|True|Port, e.g. 389|None|389|
+|use_ssl|boolean|None|True|Use SSL?|None|True|
+|username_password|credential_username_password|None|True|Username and password|None|{“username”:”user1”, “password”:”mypassword”}|
 
-The connection configuration accepts the following parameters:
+Example input:
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|use_ssl|boolean|None|True|Use SSL?|None|
-|host|string|None|True|Server Host, e.g. ldap://192.5.5.5. Must use either ldap:// or ldaps:// for SSL prefix|None|
-|port|integer|389|True|Port, e.g. 389|None|
-|username_password|credential_username_password|None|True|Username and password|None|
+```
+{
+  "host": "ldaps://198.51.100.100",
+  "port": 389,
+  "use_ssl": "true",
+  "username_password": "{“username”:”user1”, “password”:”mypassword”}"
```

Comment:
This is an object, having quotes around it makes it a string. Looks like there's unicode here too, not ASCII `"`.